recipes/recipes_emscripten/protobuf/recipe.yaml

```diff
@@ -0,0 +1,13 @@
+context:
+  version: 4.22.3
+
+package:
+  name: protobuf
+  version: "{{ version }}"
+
+build:
+  number: 0
+
+source:
```

Comment:
curious why we don't need any of the `build` requirements from https://github.com/conda-forge/protobuf-feedstock/blob/main/recipe/meta.yaml#L32-L33 ?